src/app/service/poll.service.ts

```diff
@@ -39,21 +43,32 @@ export class PollService {
    * @param id Id of the affected poll
    */
   get(id: number): Observable<Poll> {
-    return this.http.get<Poll>(ENDPOINT_URL + `/${id}`, { withCredentials: true });
+    return this.http.get<Poll>(ENDPOINT_URL + `/${id}`, {withCredentials: true});
   }
 
   /**
    * Retrieves all polls of a specific user from the server
    */
   getAll(): Observable<Poll[]> {
-    return this.http.get<Poll[]>(ENDPOINT_URL, { withCredentials: true });
+    return this.http.get<Poll[]>(ENDPOINT_URL, {withCredentials: true});
   }
 
   /**
    * Retrieves all poll items of a specific poll from the server
    */
   getAllItems(pollId: number): Observable<PollItem[]> {
-    return this.http.get<PollItem[]>(ENDPOINT_URL + `/${pollId}/poll-items`, { withCredentials: true });
+    return new Observable(subscriber => {
+      this.http.get<object[]>(ENDPOINT_URL + `/${pollId}/poll-items`, {withCredentials: true}).subscribe(pollItems => {
```

Comment:
Codacy has a fix for the issue: [Expected parentheses around arrow function argument.](https://app.codacy.com/gh/livepoll/live-poll/pullRequest?prid=7591734)
```suggestion
      this.http.get<object[]>(ENDPOINT_URL + `/${pollId}/poll-items`, {withCredentials: true}).subscribe((pollItems) => {
```